### PR TITLE
add new interactive examples

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

**Comment by fiji-flo** [comment]
It'll need a certificate for `interactive-examples.stage.mdn.mozilla.net` and at some point for `interactive-examples.prod.mdn.mozilla.net`

File: apps/mdn/mdn-aws/infra/acm.tf [hunk 1]
```diff
@@ -29,6 +29,11 @@ data "aws_acm_certificate" "prod-media-cdn-cert" {
   domain   = "media.prod.mdn.mozit.cloud"
 }
 
+data "aws_acm_certificate" "interactive_examples_stage_mdn_mozit_cloud" {
+  provider = aws.acm
+  domain   = "interactive-examples.stage.mdn.mozilla.net"
```
Comment (bkochendorfer) [inline apps/mdn/mdn-aws/infra/acm.tf:34]: This doesn't seem to be a public domain? Could we make it on the mozit.cloud or mozilla/allizom domain?

**Comment by bkochendorfer** [comment]
Did you get everything looking good here @fiji-flo ? The terraform looks good to me

**Comment by fiji-flo** [comment]
I didn't set it up for prod yet but stage works fine! We're good. Thank you.

**Pull request merged**